sdc/crypto/scripts/generate_secrets.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import os
+import yaml
+
+import hashlib
+
+from yaml.representer import SafeRepresenter
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives.serialization import load_pem_private_key, Encoding, PublicFormat
+
+
+# This script generates a yml file in the following format.
```

Comment:
I agree with the comments, i'll change to be '''

Though this isn't really a script anymore it's now more a bunch of helper methods used by this script:

https://github.com/ONSdigital/sdx-decrypt/pull/66/files#diff-d43168ad6ca7fad18a3fb9bfd90dffe3

Maybe I'll rename the file